### ISSUE TITLE
Function for easy setup of notebooks

### ISSUE THREAD

**Issue by xblahoud** [FiMDP/FiMDPEnv#2]
Currently, we need to include the following at the beginning of notebooks for the animations to be displayed nicely.
```python
from matplotlib import animation, rc
rc('animation', html='jshtml')
```
Let's do a function `setup` that does this (or something else in the future). The intended use can be something like this:
```python
import fimdpenv
fimdpenv.setup()
```
This solution is inspired by [Spot](spot.lrde.epita.fr/), where they use it to set some sane default settings for visualizations. The advantage is that this can be easily documented and we can add more setup features easily in the future.

**Comment by pthangeda** [comment]
This is a great idea! However, `fimdpenv.setup()` might not be feasible as the function has to be in a module. We can have something on the lines of `fimdpenv.setup.setup()` where the setup function exists in a module name setup. I was thinking something like this: `fimdpenv.setup.uuvenv_setup()` where the function `uuvenv_setup()` will have the default settings for uuv environment. I could also have something like `fimdpenv.uuvenv.setup()` where I can store the default settings for UUV environment as a `setup()` function in `UUVEnv` module itself.

**Comment by xblahoud** [comment]
We can actually achieve `fimdpenv.setup()` by defining it in `__init__.py`. I would start with that. If the need arises and we have different setup routines for distinct environments, we can always create more specific functions of the form `fimdpenv.uuvenv.setup()`.

**Comment by pthangeda** [comment]
Fixed in 7d7dde195904b2710c404ebe8de3ba7973e7672f. Any further additions to default settings for different environments, if the need arises, will be taken care off later.
